package.json

```diff
@@ -1,27 +1,35 @@
 {
-  "name": "basisjs",
-  "title": "basis.js",
-  "description": "JavaScript framework to build single-page applications",
-  "version": "1.4.0-dev",
-  "main": "src/basis.js",
-  "homepage": "http://basisjs.com",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/basisjs/basisjs.git"
-  },
-  "keywords": [
-    "javascript",
-    "framework"
-  ],
-  "bugs": {
-    "url": "https://github.com/basisjs/basisjs/issues"
-  },
-  "licenses": [
-    {
-      "type": "MIT",
-      "url": "https://github.com/basisjs/basisjs/blob/master/LICENSE"
+    "name": "basisjs",
+    "title": "basis.js",
+    "description": "JavaScript framework to build single-page applications",
+    "version": "1.4.0-dev",
+    "main": "src/basis.js",
+    "homepage": "http://basisjs.com",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/basisjs/basisjs.git"
+    },
+    "keywords": [
+        "javascript",
+        "framework"
+    ],
+    "bugs": {
+        "url": "https://github.com/basisjs/basisjs/issues"
+    },
+    "licenses": [
+        {
+            "type": "MIT",
+            "url": "https://github.com/basisjs/basisjs/blob/master/LICENSE"
+        }
+    ],
+    "dependencies": {},
+    "devDependencies": {
+        "karma": "^0.12.31",
+        "karma-yatra": "git://github.com/Mavrin/karma-basisjs-test-runner.git",
+        "karma-cli": "0.0.4",
+        "karma-firefox-launcher": "^0.1.4"
+    },
+    "scripts": {
+        "test": "./node_modules/.bin/karma start karma.conf.js"
```

Comment:
This command describe in http://karma-runner.github.io/0.12/plus/travis.html. But i replace it.